README.md

```diff
@@ -71,10 +68,10 @@ jupyter-book build example-book/ --builder html
 
 ### Deployment
 
-For more context, see the [Jupyter Book GitHub Pages Guide](https://jupyterbook.org/en/stable/publish/gh-pages.html).
+We want to [publish a GitHub Pages site using a GitHub Actions workflow](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow). See also the [Jupyter Book GitHub Pages Guide](https://jupyterbook.org/en/stable/publish/gh-pages.html).
```

Comment:
Add link to official GitHub Pages docs, to provide the user with more context, if they are unfamiliar